tests/python/sdk/test_auto_annotation.py

```diff
@@ -1038,6 +1089,29 @@ def test_attribute_val_unsuitable_for_spec(self):
     import torch
     import torch.nn as nn
 
+    class FakeTorchvisionClassifier(nn.Module):
+        def __init__(self, label_id: int) -> None:
+            super().__init__()
+            self._label_id = label_id
+
+        def forward(self, images: torch.Tensor) -> torch.Tensor:
+            assert isinstance(images, torch.Tensor)
+
+            probs = [0.0] * 1000
+            # Since the function does a softmax operation over the output,
+            # we can't test edge cases by making the final probability exactly 0.75.
+            # With this value the final probability is ~0.77.
```

Comment:
I guess you could test the threshold application by returning 2 different output vectors for 2 images:

image1: output = [<threshold, 0, ...] -> no annotations
image2: output = [>threshold, 0, ...] -> a tag with label

Reply:
The code is fine, my comment is only about the line

> we can't test edge cases by making the final probability exactly 0.75.

Reply:
Okay. So do you think I should change something?